config/initializers/aws.rb

```diff
@@ -0,0 +1,9 @@
+Aws.config.update(
+  region: 'us-east-1',
+  endpoint: 'http://localhost:9000',
```

Comment:
This should use docker-compose links (exposed as ENV vars), because minio won't be always running on localhost:9000 ;)
Please check how DB is configured and do the same for storage service.
Something like
```
    links:
      - storage
    environment:
      TVARKAU_VILNIU_RAILS_STORAGE_HOST: storage
      TVARKAU_VILNIU_RAILS_STORAGE_PORT: 9000
```
and then as in `database.yml`:
```
host: <%= ENV['TVARKAU_VILNIU_RAILS_STORAGE_HOST'] || 'localhost' %>
port: <%= ENV['TVARKAU_VILNIU_RAILS_STORAGE_PORT'] || '9000' %>
```